api/openapi.yaml

```diff
@@ -185,6 +185,14 @@ paths:
         - "Users"
       description: A user can log out from his/her account.
       summary: "Log out from account"
+      requestBody:
+        required: true
+        content:
```

Comment:
if it is the id of user you try to log out, it is better to put it as a path param to keep it consistent with the other methods.

Reply:
changes are done. Now user id is in path param